core/templates/core/downloads.html

```diff
@@ -0,0 +1,27 @@
+
+{% extends 'core/base.html' %}
+
+{% load static %}
+
+{% block title %}
+  Squire - Downloads
+{% endblock title %}
+
+{% block og-title %}
+    Knights Downloads
+{% endblock og-title %}
+{% block og-description %}
+    A central location for association protocols or documents.
+{% endblock og-description %}
+
+{% block content %}
+    <img src="{% static 'images/newsletter_header.png' %}" alt="Knights newsletter header image" width="100%" />
```

Comment:
As this now also used in downloads, rename the file to association_header.png or something like that